docs/core/building/distribution-packaging.md

```diff
@@ -0,0 +1,69 @@
+---
+title: .NET Core distribution packaging
+description: .NET Core distribution packaging
+keywords: .NET, .NET Core, source, build
+author: beleroy
+ms.author: mairaw
+ms.date: 03/29/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: 71b9d722-c5a8-4271-9ce1-d87e7ae2494d
+---
+
+.NET Core distribution packaging
+================================
+
+As .NET Core becomes available on more and more platforms, thanks to the efforts of dedicated contributors, it is useful to provide guidelines on how to package, name, and version it, in order to ensure a consistency of experience no matter where you choose to run .NET.
```

Comment:
thanks to the efforts of dedicated contributors seems like content that would be in a blog post not on official docs

Suggested rewriting:
As .NET Core becomes available on more and more platforms, it's useful to learn how to package, name, and version it, in order to ensure a consistent experience no matter where you choose to run .NET.